pkg/bot/interactive/msg_test.go

```diff
@@ -0,0 +1,37 @@
+package interactive
+
+import (
+	"fmt"
+	"testing"
+
+	"gotest.tools/v3/golden"
+)
+
+// go test -run=TestInteractiveMessageToMarkdown ./pkg/bot/interactive/... -test.update-golden
+func TestInteractiveMessageToMarkdown(t *testing.T) {
+	tests := []struct {
+		name    string
+		lineFmt func(msg string) string
+	}{
+		{
+			name:    "render with Markdown new lines",
```

Comment:
Right? 😄 
```suggestion
			name:    "render with MS Teams new lines",
```

Reply:
![homer-simpson-hide-in-shrubs](https://user-images.githubusercontent.com/17568639/191011061-c26caa26-2779-42f7-bb3b-c9ffcb9caf00.gif)